doc/manual/install-judgehost.rst

```diff
@@ -39,8 +39,8 @@ For Debian::
 
 For RHEL 7/Fedora [*]_::
 
-  sudo yum install make pkgconfig sudo libcgroup-devel lsof \
-        php-cli php-mbstring php-xml php-process procps-ng gcc g++
+  sudo dnf install make pkgconfig sudo libcgroup-devel lsof \
```

Comment:
Renaming to `dnf` here would break RHEL7 but the people still using that should know how to change this.

Reply:
I am fine with removing the explicit reference to RHEL here, or noting that this would only be for 8.

Reply:
No, I think this is fine. Any university or company using RHEL7 should have gone to RHEL8 now (https://www.redhat.com/en/blog/end-maintenance-red-hat-enterprise-linux-7-almost-here)